python_modules/dagster/dagster/_utils/error.py

```diff
@@ -93,6 +97,45 @@ def _should_redact_user_code_error() -> bool:
     "DAGSTER_REDACTED_ERROR_LOGGER_NAME", "dagster.redacted_errors"
 )
 
+error_id_by_exception: ContextVar[Mapping[int, str]] = ContextVar(
+    "error_id_by_exception", default={}
+)
```

Comment:
the name here should probably have 'redacted' in it: redacted_user_code_error_id_by_exception?